manifests/gcp_marketplace/guide.md

```diff
@@ -59,13 +59,10 @@ and store the service account credential as a Kubernetes secret `user-gcp-sa` in
 # Create credential for the service account
 gcloud iam service-accounts keys create application_default_credentials.json --iam-account $SA_NAME@$PROJECT_ID.iam.gserviceaccount.com
```

Comment:
gcloud iam service-accounts keys create application_default_credentials.json \
  --iam-account $SA_NAME@$PROJECT_ID.iam.gserviceaccount.com

how about also split it into 2 lines for easy copy?

Reply:
Done